lib/phlex/heroicons/base.rb

```diff
@@ -3,10 +3,10 @@
 # rubocop:disable Lint/MissingSuper
 module Phlex
   module Heroicons
-    class Base < Phlex::View
+    class Base < Phlex::SVG
       attr_reader :variant, :classes
 
-      def initialize(variant: Phlex::Heroicons.config.default_variant, classes: Phlex::Heroicons.config.default_class)
+      def initialize(variant: Phlex::Heroicons.configuration.default_variant, classes: Phlex::Heroicons.configuration.default_class)
```

Comment:
I didn't notice `config` defined but `configuration` is defined.

Reply:
This was needed when I pointed my Gemfile to my local directory, but when I point it to this GitHub branch, this doesn't seem to work. Might need to revisit this part of the diff.